src/api/python/cvc5.pxi

```diff
@@ -251,16 +277,26 @@ cdef class DatatypeConstructor:
 
 
 cdef class DatatypeConstructorDecl:
+    """Wrapper class for :cpp:class:`cvc5::api::DatatypeConstructorDecl`."""
     cdef c_DatatypeConstructorDecl cddc
     cdef Solver solver
 
     def __cinit__(self, Solver solver):
         self.solver = solver
 
     def addSelector(self, str name, Sort sort):
+        """
+            Add datatype selector declaration.
+            :param name: the name of the datatype selector declaration to add.
+            :param sort: the range sort of the datatype selector declaration to add.
```

Comment:
Not sure what is wrong here, but the parameters are not properly rendered. (also in other places).